### DISCUSSION TITLE
Configurable Production

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#1961] [Questions & Answers]
Request:
is there a way to sequentially enable configurable product options?

example t-shirt 2 options:
1 color
2 size

What I want:
if point 1 is not selected, point 2 is not enabled.

**Comment by f1-outsourcing** [comment]
What is wrong with making both required?

**Reply by Danieleeffe1** [reply]
hello and thank you for your interest

I use google translate so I apologize if I can't explain myself well or if I misunderstand what you write.

Both must be mandatory but point 2 is bound to the choice of point one.

if point 1 is not selected, point 2 should not give the possibility to choose.

I have seen that in some ecommerce (I don't know which ones) this happens in Magento / OpenMage I don't see this possibility.

**Reply by f1-outsourcing** [reply]
No worries about language.  Configurable options that are depending on each other are not supported by default. So you have to go for some module or so. From what I have quickly read is, that the most common advice is, to work around this by using bundled products. 

I have been working on a api call that is being done when custom options are chosen, so the customer just chooses, and before it is added to the cart, it is remotely checked if it is available. That could also be a solution. (but you have to maintain some external api/db with up to date information)
